docs/en/reference/sql/dql/SELECT_STATEMENT.md

```diff
@@ -111,27 +111,21 @@ TableAsName
+| [`HAVING` Clause](../dql/HAVING_CLAUSE.md)     | **``✓``**    |                     |                     | The Having clause is similar to the Where clause. The Having clause filters data after GroupBy, and the Where clause is used to filter records before aggregation.                                                                                                                                                                                                                      |                                                                                                                                                                                                                                                                                                                                                                              |
+| [`WINDOW` Clause](../dql/WINDOW_CLAUSE.md)     | **``✓``**    |                     | **``✓``**           | The window clause is used to define one or several windows. Windows can be named or anonymous. Users can call aggregate functions on the window to perform analysis (```sql agg_func() over window_name```). During Online Serving, please follow the [specification of WINDOW Clause under Online Serving](../deployment_manage/ONLINE_SERVING_REQUIREMENTS.html#online-servingwindow) |
+| [`LIMIT` Clause](../dql/LIMIT_CLAUSE.md)       | **``✓``**    | **``✓``**           | **``✓``**           | The Limit clause is used to limit the number of results. OpenMLDB currently only supports one parameter to limit the maximum number of rows of returned data.                                                                                                                                                                                                                           |
+| `ORDER BY` Clause                              |              |                     |                     | Standard SQL also supports the OrderBy clause, however OpenMLDB does not support this clause currently. For example, the query `SELECT * from t1 ORDER BY col1;` is not supported in OpenMLDB.                                                                                                                                                                                          |
 
 ```{warning}
-The online mode or the stand-alone version of the selection may not obtain complete data.
-Because a query may perform a large number of scans on multiple tablet servers, for the stability of tablet servers, a single tablet server limits the maximum amount of scanned data, namely `scan_max_bytes_size`.
+The `SELECT` running in online mode or the stand-alone version may not obtain complete data.
+Because a query may perform a large number of scans on multiple tablets, for stability, the maximum scanned data on one tablet has been limited, namely `scan_max_bytes_size`.
```

Comment:
the maximum scanned data on one tablet has been limited
the largest number of bytes to scan is limited